zk/stages/stage_sequence_execute_utils.go

```diff
@@ -282,6 +297,10 @@ func prepareL1AndInfoTreeRelatedStuff(sdb *stageDb, batchState *BatchState, prop
 		ger = l1TreeUpdate.GER
 	}
 
+	if batchState.isResequence() && l1TreeUpdateIndex == 0 {
+		shouldWriteGerToContract = false
+	}
```

Comment:
I don't think we need this, if there is no change in the l1 info tree index we won't write the GER anyway.

Reply:
Agreed. Will remove.